components/dictionary.txt

```diff
@@ -1,11 +1,133 @@
+actin
+ADT
+al
 al.
+AML
+AnaLysis
+ASXL
+Azacytidine
+arounds
+Baumer
+Benjamini
+bioconductor
+bioinformatics
+Brems
+CCDL
+CCDL's
+cheatsheet
+cheatsheets
+ChIP
+Cmd
+ColorBrewer
+ComplexHeatmap
+CREB
+Crouser
+cytosolic
+Danio
 dataset
 dataset's
+decitabine
 DESeq
 DESeq2
+DESeqDataset
+devtools
+DGE
+directionality
+DocToc
+Dorsoventral
+ECM
+edgeR
+edgeR's
+eg
+ENSDARG
+Ensembl
+ENSG
+Entrez
+et
 et.
+FACS
+functionalize
+FPKM
+GEne
+genesets
+generalizable
+ggplot
+GitHub
+glioma
+GC
+GSE
+GSEA
+HCOP
+hexamer
+HGNC
+histological
+Hochberg
+hypomethylating
+IDH
+Illumina
+IRF
+isoform
+isoforms
+jpeg
+KEGG
+limma
+logFC
+medulloblastoma
+molecularly
+musculus
+MSigDB
+myeloid
+Northcott
+nb
+ortholog
+orthologs
+orthology
+overexpressing
+overexpression
+pheatmap
+PLoS
+PLX
+PNAS
 PNG
+polymerase
+probesets
+prostatectomy
+PRPS
+pre
 `.Rmd`
+QuSAGE
+qusage
+QN
+README
+rerio
+ribosomes
+RMA
+RPKMs
 RStudio
+SeT
+ssGSEA
+StatQuest
+SuperSeries
+str
+subtype
+subtypes
 tidyverse
+TPM
+TPMs
+transcriptional
+transcriptome
+transcriptomic
 TSV
+tximport
+ucture
```

Comment:
Is this correct?

Reply:
See my comment above about: https://github.com/AlexsLemonade/refinebio-examples/blob/e630a4eec112271038f7c1a1a645f533a119fb3b/02-microarray/pathway_analysis_microarray_02_ora_with_webgestaltr.Rmd#L132 I hate it but don't know what else to do about it.

Reply:
I think its unlikely for this misspelling to occur naturally so I was okay with this add.